openadapt/events.py

```diff
@@ -296,30 +351,22 @@ def get_timestamp_mappings(to_merge):
                     dx = abs(event.mouse_x - prev_pressed_event.mouse_x)
                     dy = abs(event.mouse_y - prev_pressed_event.mouse_y)
                     if (
-                        dt <= double_click_interval and
-                        dx <= double_click_distance and
-                        dy <= double_click_distance
+                        dt <= double_click_interval
+                        and dx <= double_click_distance
+                        and dy <= double_click_distance
                     ):
-                        press_to_press_t[prev_pressed_event.timestamp] = (
-                            event.timestamp
-                        )
+                        press_to_press_t[prev_pressed_event.timestamp] = event.timestamp
                 prev_pressed_event = event
             elif prev_pressed_event:
                 if prev_pressed_event.timestamp in press_to_release_t:
                     # should never happen
                     logger.warning("consecutive mouse release events")
-                press_to_release_t[prev_pressed_event.timestamp] = (
-                    event.timestamp
-                )
+                press_to_release_t[prev_pressed_event.timestamp] = event.timestamp
         return press_to_press_t, press_to_release_t
 
-
-    def get_merged_events(to_merge, state):
+    def get_merged_events(to_merge: Any, state: Any) -> list:
```

Comment:
`to_merge: list[ActionEvent], state: dict[str, Any]) -> list[ActionEvent]`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507